Src/VsSpecific/Vs2017/Vs2017.csproj

```diff
@@ -4,10 +4,16 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Vim.VisualStudio.Vs2017</RootNamespace>
     <AssemblyName>Vim.VisualStudio.Vs2017</AssemblyName>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>net461</TargetFramework>

```

Comment:
Don't think this should be necessary. The Microsoft.CodeAnalysis.CSharp.Scripting library targets netstandard1.3 which is compatible with net46.